### PR TITLE
Add rollout rate support for the publish to testing

### PR DESCRIPTION
**Pull request by LawrenceHwang**
We need to be able to publish to all zones without rollout policy. This PR enables the support to specify rollout rate in the workflows and task. The publish to testing workflows have the rollout rate set to `0`.

### CODE REVIEW

File: concourse/tasks/daisy-publish-images.yaml [hunk 1]
```diff
@@ -12,13 +12,15 @@ inputs:
...
   GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+  ROLLOUT_RATE: 60
 
 run:
   path: /gce_image_publish
   args:
   - -source_gcs_path=((source_gcs_path))
   - -source_version=((source_version))
   - -publish_version=((publish_version))
+  - -rollout_rate=$ROLLOUT_RATE
```
Comment (hopkiw) [inline concourse/tasks/daisy-publish-images.yaml:23]: this form of referencing environment variables is a shell feature. this container doesn't run a shell, it runs the binary `/gce_image_publish`. so this string will be read literally.

what is your testing mechanism for this change?

File: concourse/pipelines/windows-image-build.yaml [hunk 2]
```diff
@@ -3829,6 +3907,8 @@ jobs:
     file: publish-version/version # produced from generate-version task
   - task: publish-2019-for-containers
     file: guest-test-infra/concourse/tasks/daisy-publish-images.yaml
+    params:
+      ROLLOUT_RATE: 0
```
Comment (hopkiw) [inline concourse/pipelines/windows-image-build.yaml:3911]: i see you're setting this to 0 in many places, should the default be zero?

**Review by hopkiw — CHANGES_REQUESTED** [review]
my understanding is rollout policies can be stored in the image publish workflows themselves, have we considered that option?

File: concourse/pipelines/windows-image-build.yaml (hunk 2 above)
Comment (LawrenceHwang) [reply concourse/pipelines/windows-image-build.yaml]: 0 is desired value for those “publish to testing” workflows. The other publish jobs (to staging and prod) should still uses 60.

File: concourse/tasks/daisy-publish-images.yaml (hunk 1 above)
Comment (LawrenceHwang) [reply concourse/tasks/daisy-publish-images.yaml]: Thank you for the feedback and pointers. I've updated the PR with new task in this case.

**Comment by LawrenceHwang** [comment]
> my understanding is rollout policies can be stored in the image publish workflows themselves, have we considered that option?

Thank for sharing the historical context. As chatted, the current implementation doesn't support describing rollout policies in the workflow. Will create a tracking bug.

**Review by hopkiw — APPROVED** [review]

**Pull request merged**